rbac/server/db/proposals_query.py

```diff
@@ -191,16 +191,17 @@ def fetch_open_proposals_by_opener(next_id):
     return resource
 
 
-def fetch_open_proposals_by_target(next_id):
+async def fetch_open_proposals_by_target(conn, next_id):
     """Fetch all open proposals where target is the given next_id.
     Args:
         next_id:
             str: a user's next ID
     """
     resource = (
-        r.table("proposals")
+        await r.table("proposals")
         .filter({"target_id": next_id, "status": "OPEN"})
```

Comment:
After looking at the proposal table, proposal entries do not have `target_id` field on it.

They have two other fields that you'll need to use when looking at role relationship proposals:
object_id: The field for the role_id where the user will be added to
related_id: The field for the next_id that will be added to the role